gradle/plugins/src/main/kotlin/com.hedera.gradlebuild.jpms-modules.gradle.kts

```diff
@@ -50,86 +42,77 @@ dependencies.components {
             "org.codehaus.mojo:animal-sniffer-annotations"
         )
 
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-netty") {
-        params(grpcComponents + annotationLibraries)
-    }
-    withModule<AddDependenciesMetadataRule>("io.grpc:grpc-netty") { params(grpcModule) }
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-protobuf") {
-        params(grpcComponents + annotationLibraries)
-    }
-    withModule<AddDependenciesMetadataRule>("io.grpc:grpc-protobuf") { params(grpcModule) }
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-protobuf-lite") {
-        params(
-            grpcComponents + annotationLibraries + listOf("com.google.protobuf:protobuf-javalite")
-        )
+    module("io.netty:netty-transport-native-epoll") {
+        addFeature("linux-x86_64")
+        addFeature("linux-aarch_64")
     }
-    withModule<AddDependenciesMetadataRule>("io.grpc:grpc-protobuf-lite") {
-        params(grpcModule + listOf("com.google.protobuf:protobuf-java"))
+    module("io.grpc:grpc-netty") {
+        annotationLibraries.forEach { removeDependency(it) }
```

Comment:
This file looks way more complex than the last time I looked at it. We need to create shared knowledge on the patterns and apis.

Reply:
When was that last time? From my perception it became more and more readable with each iteration. 😄 But there was also a time where it was shorter before **everything** was patched to be a module.

I agree on the knowledge sharing though. In the IDE you can now click on the "methods" used here and get some documentation on what they are doing. And there are some comments in the file. But we can add more. Maybe this is something we should have a chat about to figure out what's the best way is to improve this. (in a follow up to this PR)